### PR TITLE
fix(ratings): UTC timezone, formatting and segments

### PR DESCRIPTION
**Pull request by williambsm**

### CODE REVIEW

File: express/blocks/ratings/ratings.js [hunk 1]
```diff
@@ -124,12 +125,12 @@ export default function decorate($block) {
...
-          name: 'Timestamp',
-          value: new Date().toLocaleString(),
+          name: 'Feedback',
+          value: comment,
         },
         {
-          name: 'Comment',
-          value: comment,
+          name: 'Timestamp',
+          value: new Date().toLocaleString('en-US', { timeZone: 'UTC' }),
```
Comment (rofe) [inline express/blocks/ratings/ratings.js:133]: why not simply `new Date().toUTCString()`?

File: express/blocks/ratings/ratings.js [hunk 2]
```diff
@@ -109,11 +109,12 @@ export default function decorate($block) {
   }
 
   function submitRating(rating, comment) {
+    const segments = Context.get('segments');
     const content = {
       data: [
         {
           name: 'Segments',
-          value: Context.get('audiences') ?? '',
+          value: segments.length ? segments.join(', ') : '',
```
Comment (rofe) [inline express/blocks/ratings/ratings.js:117]: Does this filter the relevant segments only?

**Review by rofe — CHANGES_REQUESTED** [review]

File: express/blocks/ratings/ratings.js (hunk 1 above)
Comment (williambsm) [reply express/blocks/ratings/ratings.js:133]: We can do, but I found this to be more readable. I'll make the change!

Comment (rofe) [reply express/blocks/ratings/ratings.js:133]: OK let's leave it then

**Review by rofe — APPROVED** [review]

**Pull request merged**